docker/Dockerfile-windows-java-base

```diff
@@ -18,12 +18,12 @@ ENV ProgramFiles="C:\Program Files"
 ENV WindowsPATH="C:\Windows\system32;C:\Windows"
 
 ARG USER_HOME_DIR="C:/Users/ContainerUser"
-ARG MAVEN_VERSION=3.8.3
-ARG SHA=959de0db3e342ecf1c183b321799a836c3c10738126f3302b623376efa45c6769ccb5cd32a17f9a9a8eb64bb30f13a6a0e4170bf03a7707cfba6d41392107e93
+ARG MAVEN_VERSION=3.8.5
```

Comment:
Had to update Maven because version 3.8.3 was removed from the mirror and the build was failing then.